README.md

```diff
@@ -88,31 +88,49 @@ Flutter Shortcuts allows you to create shortcut icon from both android `drawable
 
 * If you want to use icon from Android resources, `drawable` or `mipmap`.
 
-use: `ShortcutIconAsset.androidAsset`
+  use: `ShortcutIconAsset.androidAsset`
 
-```dart
-ShortcutItem(
-  id: "2",
-  action: 'Bookmark page action',
-  shortLabel: 'Bookmark Page',
-  icon: "ic_launcher",
-  shortcutIconAsset: ShortcutIconAsset.androidAsset,
-),
+  ```dart
+  ShortcutItem(
+    id: "2",
+    action: 'Bookmark page action',
+    shortLabel: 'Bookmark Page',
+    icon: "ic_launcher",
+    shortcutIconAsset: ShortcutIconAsset.androidAsset,
+  ),
 ```
 
 * If you want to create shortcut icon from flutter asset. (DEFAULT)
 
-use: `ShortcutIconAsset.flutterAsset`
+  use: `ShortcutIconAsset.flutterAsset`
```

Comment:
The whole section is converted to the code block. The reason may be the space behind `use: ...`.